apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2/BigtableClientPartial.cs

```diff
@@ -419,6 +419,220 @@ public partial class BigtableClient
         }
 
         /// <summary>
+        /// Modifies a row atomically. The method reads the latest existing version
```

Comment:
This sounds like the client is doing the reading then writing, whereas in reality it's server-side, right?

Reply:
Correct. Note that these are just the auto-generated docs copied here, so if someone uses the generated overloads, they're still going to see this summary. But I can add "...on the server." to the end of this sentence to clear up these overloads.